javascript/src/openapi/http/http.ts

```
@@ -27,9 +26,9 @@ export type HttpFile = Blob & { readonly name: string };
 
 
 export class HttpException extends Error {
-    public constructor(msg: string) {
-        super(msg);
-    }
+  public constructor(msg: string) {
+    super(msg);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)